zypp/base/IOTools.cc

```diff
@@ -0,0 +1,129 @@
+/*---------------------------------------------------------------------\
+|                          ____ _   __ __ ___                          |
+|                         |__  / \ / / . \ . \                         |
+|                           / / \ V /|  _/  _/                         |
+|                          / /__ | | | | | |                           |
+|                         /_____||_| |_| |_|                           |
+|                                                                      |
+\---------------------------------------------------------------------*/
+/** \file	zypp/base/IOTools.cc
+ *
+*/
+
+#include <errno.h>
+#include <fcntl.h>
+#include <iostream>
+#include <glib.h>
+
+#include <zypp/AutoDispose.h>
+#include <zypp/base/IOTools.h>
+#include <zypp/base/Logger.h>
```

Comment:
JFYI include zypp/base/LogTools.h if you want to dump container types.

Reply:
Done